nifi-nar-bundles/nifi-site-to-site-reporting-bundle/nifi-site-to-site-reporting-task/src/main/java/org/apache/nifi/reporting/SiteToSiteProvenanceReportingTask.java

```diff
@@ -67,14 +73,73 @@
 
     static final PropertyDescriptor PLATFORM = new PropertyDescriptor.Builder()
         .name("Platform")
+        .displayName("Platform")
         .description("The value to use for the platform field in each provenance event.")
         .required(true)
         .expressionLanguageSupported(true)
         .defaultValue("nifi")
         .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
         .build();
 
+    static final PropertyDescriptor FILTER_EVENT_TYPE = new PropertyDescriptor.Builder()
```

Comment:
These newly added properties are not added to `getSupportedPropertyDescriptors` method. So I was not able to configure filters from NIFI UI.

Since this is a simple fix, I went ahead and added these filters and able to confirm provenance events are filtered. LGTM, +1. I will merge this into master. Thanks @pvillard31 !